src/pages/pull/read/components/ReviewerAdd.tsx

```diff
@@ -64,7 +65,7 @@ export default function ReviewerAdd() {
                                 reviewers.indexOf(address) > -1 ? 'opacity-100' : 'opacity-0'
                               }`}
                             />
-                            {address}
+                            {shortenAddress(address, 6)}
```

Comment:
Does it have a way to see the full address? Like a tooltip when you hover for a few seconds?